src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -102,7 +103,7 @@ export const RuntimeTransactionDetailPage: FC = () => {
     data?.data,
   )
 
-  const tokenPriceInfo = useTokenPrice(getTickerForNetwork(scope.network))
+  const tokenPriceInfo = useTokenPrice(getTickerForScope(scope))
```

Comment:
I would rethink this. Do we want to fetch token price for some other coin? Not sure if we are paying for CoinGecko API, or is it rate limited? Maybe @buberdds knows more

And there is not even a guarantee, that the coin is listed. It will just error out. So there needs to be a consideration of removing the price widget, in case of non ROSE token.

Reply:
Also removing every token price information through the application.

Reply:
Your assumptions are correct Matej. We do not pay for API and on a free plan number of requests is limited. I would not worry about limit though, but chances are very high that a token won't be avail in API.

Reply:
> Do we want to fetch token price for some other coin? 
> And there is not even a guarantee, that the coin is listed. It will just error out. 

Yes, and that is why we _don't_ actually attempt to fetch them. Here is the code doing the actual fetching:

https://github.com/oasisprotocol/explorer/blob/dc16abce4f809b9b1c8fe910088543ca0704a081/src/coin-gecko/api.ts#L55-L79

We are only ever fetching ROSE, we are faking the data for all others.

So there is no problem with the data not being available.

Reply:
There is a problem with that. In case there is no token price data, we should hide the following price widget:

![image](https://github.com/oasisprotocol/explorer/assets/9722540/6dc21ece-a54b-4ea7-afc7-1d1a51a6293c)

Also it makes no sense to show this fields:

![image](https://github.com/oasisprotocol/explorer/assets/9722540/2fd63a77-2f15-43f8-a6f2-0e3002293a58)

Reply:
>  In case there is no token price data, we should hide the following price widget:

We _are_ hiding the price widget for testnet tokens. Instead, we are advertising the faucet. See [here](https://github.com/oasisprotocol/explorer/blob/csillag/special-ticker-for-paratime/src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx#L56-L59).

Reply:
> Also it makes no sense to show this fields:
> 
> ![image](https://private-user-images.githubusercontent.com/9722540/305416628-2fd63a77-2f15-43f8-a6f2-0e3002293a58.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgwODc5MjMsIm5iZiI6MTcwODA4NzYyMywicGF0aCI6Ii85NzIyNTQwLzMwNTQxNjYyOC0yZmQ2M2E3Ny0yZjE1LTQzZjgtYTZmMi0wZTMwMDIyOTNhNTgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMTZUMTI0NzAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGQ4MmY5ZjdmNjA1NjUzNDA4ZTY1ZjI2ODk4MjAyMjNmYmI1Y2M2NjY0MDU1M2ZhN2QyMGYxOTdjZmQwMzRiNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.WXjjyMMZdLoj_1b_WUH5D3C1EpzEgxGyCABe5g_8UoA)

Right, and that's why we are also hiding that one, see [here](https://github.com/oasisprotocol/explorer/blob/csillag/special-ticker-for-paratime/src/app/pages/RuntimeTransactionDetailPage/index.tsx#L329-L340).

As far as I can tell, this works as intended.

Reply:
(I have now updated the faucet code to better handle the new, par-ParaTime token configuration, but even the original code was smart enough not to show rose price for the testnet when we only have test tokens)

Reply:
Ok, got it. I see it now 👍 Thanks for detailed explanation.